### PR TITLE
TW 532 Removing a user from a team vs removing a paid slot

### PR DESCRIPTION
**Pull request by HeatherStenson**

### CODE REVIEW

**Review by taehoshino — APPROVED** [review]
Can you please add Kalpana to review?

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 1]
```diff
@@ -90,7 +90,9 @@ To confirm, select **Remove User From Team**.
...
 
-> When you remove a user from your team, you will still retain access to any data they have shared with the team. However, any data in the user’s personal workspaces is owned by that user and their ownership for that data won't be transferred when they're removed from your team.
+When you remove a user from your team, you will still retain access to any data they have shared with the team. However, any data in the user’s personal workspaces is owned by that user and their ownership for that data won't be transferred when they're removed from your team.
+
+> **Billing ramifications:** When you remove a member from your team, this doesn't automatically reduce the number of paid slots for your team unless the member was added by [Auto-Flex](/docs/administration/billing/#utilizing-auto-flex) during the current Auto-Flex cycle. If you want to reduce the number of paid slots on your team, a member of your team with the [Billing role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) must [edit your plan](/docs/administration/billing/#changing-your-plan).
```
Comment (Kalpana-Postman) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:95]: Can we replace billing ramifications? We do not want to call attention to billing here.
Please note: When you remove a member from your team, this doesn't automatically reduce the number of paid slots for your team unless the member was added by [Auto-Flex](/docs/administration/billing/#utilizing-auto-flex) during the current Auto-Flex cycle. If you would like to reduce the number of paid slots on your team, a member of your team with the [Billing role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) can [edit your plan](/docs/administration/billing/#changing-your-plan).

**Review by claireriley — APPROVED** [review]
LGTM! ☀️

File: src/pages/docs/administration/managing-your-team/managing-your-team.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/administration/managing-your-team/managing-your-team.md]: Thanks @Kalpana-Postman! Updated.

**Pull request merged**